proxyd/cache.go

```diff
@@ -101,9 +109,9 @@ func (c *redisCache) Get(ctx context.Context, key string) (string, error) {
 	return val, nil
 }
 
-func (c *redisCache) Put(ctx context.Context, key string, value string) error {
+func (c *redisCache) Put(ctx context.Context, key string, value string, shortLived bool) error {
 	start := time.Now()
-	err := c.redisClient.SetEx(ctx, c.namespaced(key), value, c.ttl).Err()
+	err := c.redisClient.SetEx(ctx, c.namespaced(key), value, c.defaultTTL).Err()
```

Comment:
this should pass `shortLivedTTL` if `shortLived` is `true`?

Reply:
ah yeah, fixed that!